CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## Unreleased changes
+- Added support for GRPC V2 Get Banned Peers
```

Comment:
Please markdown' format it. I.e. `GetBannedPeers` and then a small explanation for each entry.